ADD: New event card

kherkatary opened the pull request:
### ADD: 
New Event Card component

#### Card structure
- size : small | large
- card content to be passed as props

kherkatary committed:
merge from dev

kherkatary committed:
ADD: New Event card with
new in this component
1. size= large | small to be passed in props
2.the card content to be passed as props

kherkatary committed:
Merge branch 'dev' of https://github.com/E-cell-NITS/Srijan-2023 into New-Event-Card

codadept commented:
![image](https://user-images.githubusercontent.com/63517465/222978108-b924ec3c-35c6-499a-8d7e-e129f624fa03.png)

- Increase the font size of register and details buttons a bit.,

- Also if the old EventCard Component won't be used again then delete that component and rename the NewCard Component to EventCard Component

codadept reviewed (changes requested):
Checkout the comment for requested changes

codadept commented:
Also properly display the Event Card component in the Home Page

kherkatary committed:
Merge branch 'dev' of https://github.com/E-cell-NITS/Srijan-2023 into New-Event-Card

kherkatary committed:
register font++,

kherkatary committed:
FIX:
1. Deletion of old event card
2.Font size++ on register buttons

kherkatary committed:
FIX: 1. display of event cards in homepage
2.display of cards to inline-block

JOSU9435 commented on src/Pages/Home/Home.jsx:
```diff
@@ -9,16 +9,63 @@ import {
+        title2={dummy[0].title2}
+        image={dummy[0].image}
+      />
+
+      <EventCard
+        size="small"
+        id={dummy[1].id}
+        content={dummy[1].content}
+        title1={dummy[1].title1}
+        title2={dummy[1].title2}
+        image={dummy[1].image}
+      />
+
+      <EventCard
+        size="small"
+        id={dummy[2].id}
+        content={dummy[2].content}
+        title1={dummy[2].title1}
+        title2={dummy[2].title2}
+        image={dummy[2].image}
+      />
+
+      <EventCard
+        size="small"
+        id={dummy[3].id}
+        content={dummy[3].content}
+        title1={dummy[3].title1}
+        title2={dummy[3].title2}
+        image={dummy[3].image}
+      />
+
+      <EventCard
+        size="small"
+        id={dummy[0].id}
+        content={dummy[0].content}
+        title1={dummy[0].title1}
+        title2={dummy[0].title2}
+        image={dummy[0].image}
+      />
+
```
display cards using **map** function

JOSU9435 commented on src/Components/EventCard/Large.module.scss:
```diff
@@ -0,0 +1,227 @@
+      -webkit-line-clamp: 17;
+
+      @media screen and (max-width: 500px) {
+        font-size: 4vmin;
```
rem unit would be more appropriate for font sizes

JOSU9435 reviewed (changes requested):
![image](https://user-images.githubusercontent.com/77873211/223030417-a39462c8-27e7-4f19-a560-99b5c9a359c0.png)

- proportionally decrease the border radius of the smaller card with respect to the large one
- reduce the size of the title a bit such that it fits in one line
- properly implement ellipses on content overflow
- refrain from using px as units replace all the px units in CSS files with relative units like rem, %, vh ...
-

kherkatary committed:
FIX: 1.Mapped the cards in homepage 2.
border radius maintained
3. px units changed to relative units

## To be fixed:
over flow Elipsis

JOSU9435 commented on src/Components/EventCard/EventCard.jsx:
```diff
@@ -1,42 +1,47 @@
+              <p className={styles.para}>{props.content}</p>
+              <div className={styles.buttons}>
+                <button className={styles.register}>Register</button>
+                <button className={styles.details}>Details</button>
```
replace  `<button>` tag to `<Link>` tag

JOSU9435 commented on src/Components/EventCard/Large.module.scss:
replace px unit with equivalent relative unit

JOSU9435 commented on src/Components/EventCard/Small.module.scss:
replace px unit with equivalent relative unit

kherkatary committed:
Merge branch 'dev' of https://github.com/E-cell-NITS/Srijan-2023 into New-Event-Card

kherkatary committed:
FIX: 1. <Button> changed to <LINK>
2. All px changed to rem including animations
